### ISSUE TITLE
Bug: Incorrectly wrapping string month

### ISSUE THREAD

**Issue by Nixxen** [Langenscheiss/bibitnow#51]
When using the addon in my locality (Norwegian regionality, with English language settings) it fetches the name of the month as a string, instead of as a number.

When this string is wrapped in curly-braces it breaks the sorting in latex, giving this error in the latex(overleaf) error log `legacy month field 'Apr' in entry '{Langenscheiss2022Apr' is not an integer - this will probably not sort properly.`

This is from an example taken from this GitHub Issues page as the Bibtex source:
```
@misc{Langenscheiss2022Apr,
	author = {Langenscheiss},
	title = {{bibitnow}},
	journal = {GitHub},
	year = {2022},
	month = {Apr},
	note = {[Online; accessed 13. Apr. 2022]},
	url = {https://github.com/Langenscheiss/bibitnow/issues}
}
```

Removing the braces from any string will remove this issue:
```
@misc{Langenscheiss2022Apr,
	author = {Langenscheiss},
	title = {{bibitnow}},
	journal = {GitHub},
	year = {2022},
	month = Apr,
	note = {[Online; accessed 13. Apr. 2022]},
	url = {https://github.com/Langenscheiss/bibitnow/issues}
}
```

It's not much trouble fixing this manually, but it would be nice if the addon rendered the text "ready to go", so to speak.

**Comment by Langenscheiss** [comment]
Hi.
Just to be sure: it is not an option for you to have the month output in integer format? Because that option exists.
EDIT: Under Format Settings, various options, there is a checkbox "Show month in integer format".

In any case, I can add an option to use the english abbreviations for months instead. Meaning jan,feb,mar etc. but without capital letters and without quotation or anything. This is also standard bibtex as far as I know.

**Comment by Langenscheiss** [comment]
Will be fixed in 0.908. You will be able to choose between 
jan
{1}
{Jan}
{January}
in a new part of the format options panel.
The first two options should work for you. And hopefully, for those that have previously set integer format, it should automatically recognize that, as I have left the original switch to integer format as a deprecated variable in the code.

**Comment by Langenscheiss** [comment]
The month field can now be adjusted in version 0.908
